common/src/test/java/com/revenuecat/purchases/common/DeviceCacheTest.kt

```diff
@@ -64,6 +57,7 @@ class DeviceCacheTest {
 
     private val productEntitlementMappingLastUpdatedCacheKey = "com.revenuecat.purchases.api_key.productEntitlementMappingLastUpdated"
     private val productEntitlementMappingCacheKey = "com.revenuecat.purchases.api_key.productEntitlementMapping"
+    private val offeringsResponseCacheKey = "com.revenuecat.purchases.api_key.offeringsResponse"
```

Comment:
Not for this PR, but with my idea of making `DeviceCache` agnostic of what it's caching, it would be nice for this to live in `OfferingsCache`. This is at least a step in the right direction.